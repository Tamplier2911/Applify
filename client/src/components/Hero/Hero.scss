/*
.hero {
  display: grid;
  position: relative;
  overflow: hidden;
  grid-template-columns: repeat(2, 1fr);

  @media only screen and (max-width: 720px) {
    grid-template-columns: min-content;
    background-image: linear-gradient(to bottom right, #fffffffb, #ffffff2c),
      url("../../assets/jpeg/tabletbg-min.jpg");
    background-position: center;
    background-size: cover;
    background-repeat: no-repeat;
  }

  &--heading {
    grid-column: 2 / 3;
    font-family: var(--font-title);

    display: grid;
    grid-template-rows: repeat(2, min-content);
    align-content: center;
    justify-items: center;
    margin-bottom: 10rem;

    @media only screen and (max-width: 980px) {
      margin-bottom: 25rem;
      justify-items: start;
    }

    @media only screen and (max-width: 720px) {
      margin: 0;
      grid-row-gap: 4rem;
      justify-items: center;
    }

    @media only screen and (max-width: 540px) {
      grid-row-gap: 5rem;
    }

    @media only screen and (max-width: 360px) {
      grid-row-gap: 8rem;
    }

    &--wrapper {
      display: grid;
      justify-items: center;
    }

    h1 {
      font-size: 5rem;
      font-weight: 400;

      @media only screen and (max-width: 980px) {
        font-size: 4rem;
      }

      @media only screen and (max-width: 720px) {
        font-size: 4.2rem;
      }

      @media only screen and (max-width: 300px) {
        font-size: 3.5rem;
        text-align: center;
      }
    }

    h3 {
      font-weight: 400;

      @media only screen and (max-width: 980px) {
        font-size: 1.6rem;
      }

      @media only screen and (max-width: 720px) {
        font-size: 1.8rem;
      }

      @media only screen and (max-width: 300px) {
        font-size: 1.7rem;
        text-align: center;
      }
    }
  }
}

.CTA_active {
  @media only screen and (max-width: 720px) {
    display: none;
  }
}

.CTA_passive {
  @media only screen and (min-width: 721px) {
    display: none;
  }

  &:link,
  &:visited {
    text-decoration: none;
    background-color: var(--cl-tiles-2);
    padding: 1rem 5rem;
    color: var(--cl-white);
    font-size: 1.5rem;
    border: 0.1rem solid var(--cl-tiles-2);
    box-shadow: 0 0.2rem 0.4rem var(--cl-shadow);

    transition: color 0.3s, background-color 0.3s;

    @media only screen and (max-width: 540px) {
      padding: 1rem 8rem;
    }

    @media only screen and (max-width: 300px) {
      padding: 1rem 4rem;
    }
  }

  &:hover,
  &:active {
    color: var(--cl-tiles-2);
    background-color: var(--cl-white);
  }
}
*/
