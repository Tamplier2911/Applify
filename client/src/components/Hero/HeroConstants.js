const heroData = {
  eng: {
    // companyName: "Artem Nikolaiev",
    companyName: "Applify",
    companySlogan: "Creative web solutions for your business...",
    companyCTA: "Get Started",
  },
  rus: {
    // companyName: "Артём Николаев",
    companyName: "Applify",
    companySlogan: "Креативные веб-решения для вашего бизнеса...",
    companyCTA: "Начать",
  },
  ukr: {
    // companyName: "Артем Ніколаєв",
    companyName: "Applify",
    companySlogan: "Креативні веб-рішення для вашого бізнесу...",
    companyCTA: "Розпочати",
  },
};

export default heroData;
