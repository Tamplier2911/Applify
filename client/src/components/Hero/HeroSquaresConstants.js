export const squares = {
  light: [
    {
      id: 1,
      width: 10,
      height: 10,
      posX: 10,
      posY: 24,
      shadow: true,
      color: "#fe3f2d",
    },
    {
      id: 2,
      width: 10,
      height: 10,
      posX: 30,
      posY: 34,
      shadow: true,
      color: "#ffaa2d",
    },
    {
      id: 3,
      width: 10,
      height: 10,
      posX: 20,
      posY: 34,
      shadow: true,
      color: "#00a8a9",
    },
    {
      id: 4,
      width: 10,
      height: 10,
      posX: 30,
      posY: 14,
      shadow: true,
      color: "#fe3f2d",
    },
    {
      id: 5,
      width: 10,
      height: 10,
      posX: 40,
      posY: 34,
      shadow: true,
      color: "#fed595",
    },
    {
      id: 6,
      width: 5,
      height: 5,
      posX: 5,
      posY: 5,
      shadow: true,
      color: "#fe3f2d",
    },
    {
      id: 7,
      width: 10,
      height: 10,
      posX: 20,
      posY: 24,
      shadow: true,
      color: "#ffaa2d",
    },
    {
      id: 8,
      width: 5,
      height: 5,
      posX: 15,
      posY: 10,
      shadow: true,
      color: "#00a8a9",
    },
    {
      id: 9,
      width: 10,
      height: 10,
      posX: 50,
      posY: 4,
      shadow: true,
      color: "#002a38",
    },
    {
      id: 10,
      width: 10,
      height: 10,
      posX: 40,
      posY: 14,
      shadow: true,
      color: "#00a8a9",
    },
    {
      id: 11,
      width: 10,
      height: 10,
      posX: 10,
      posY: 54,
      shadow: true,
      color: "#fed595",
    },
    {
      id: 12,
      width: 10,
      height: 10,
      posX: 30,
      posY: 24,
      shadow: true,
      color: "#002a38",
    },
    {
      id: 13,
      width: 10,
      height: 10,
      posX: 20,
      posY: 44,
      shadow: true,
      color: "#fe3f2d",
    },
    {
      id: 14,
      width: 10,
      height: 10,
      posX: 10,
      posY: 44,
      shadow: true,
      color: "#ffaa2d",
    },
    {
      id: 15,
      width: 10,
      height: 10,
      posX: 40,
      posY: 4,
      shadow: true,
      color: "#fed595",
    },
    {
      id: 16,
      width: 5,
      height: 5,
      posX: 110,
      posY: 40,
      shadow: true,
      color: "#d6d7d9",
    },
    {
      id: 17,
      width: 5,
      height: 5,
      posX: 100,
      posY: 45,
      shadow: true,
      color: "#c0c1c3",
    },
    {
      id: 18,
      width: 5,
      height: 5,
      posX: 105,
      posY: 45,
      shadow: true,
      color: "#d6d7d9",
    },
    {
      id: 20,
      width: 5,
      height: 5,
      posX: 100,
      posY: 50,
      shadow: true,
      color: "#bfc0c2",
    },
    {
      id: 21,
      width: 5,
      height: 5,
      posX: 95,
      posY: 50,
      shadow: true,
      color: "#bfc0c2",
    },
    {
      id: 22,
      width: 5,
      height: 5,
      posX: 90,
      posY: 45,
      shadow: true,
      color: "#d6d7d9",
    },
    {
      id: 23,
      width: 5,
      height: 5,
      posX: 90,
      posY: 55,
      shadow: true,
      color: "#bfc0c2",
    },
    {
      id: 24,
      width: 5,
      height: 5,
      posX: 110,
      posY: 50,
      shadow: true,
      color: "#d6d7d9",
    },
    {
      id: 25,
      width: 4,
      height: 4,
      posX: 2,
      posY: 40,
      shadow: true,
      color: "#ffaa2d",
    },
    {
      id: 26,
      width: 4,
      height: 4,
      posX: 110,
      posY: 15,
      shadow: true,
      color: "#fed595",
    },
    {
      id: 27,
      width: 4,
      height: 4,
      posX: 0,
      posY: 54,
      shadow: true,
      color: "#fe3f2d",
    },
    {
      id: 28,
      width: 4,
      height: 4,
      posX: 90,
      posY: 5,
      shadow: true,
      color: "#00a8a9",
    },
  ],
  dark: [
    {
      id: 1,
      width: 10,
      height: 10,
      posX: 10,
      posY: 24,
      shadow: true,
      color: "#397e67",
    },
    {
      id: 2,
      width: 10,
      height: 10,
      posX: 30,
      posY: 34,
      shadow: true,
      color: "#225e4a",
    },
    {
      id: 3,
      width: 10,
      height: 10,
      posX: 20,
      posY: 34,
      shadow: true,
      color: "#1c6d52",
    },
    {
      id: 4,
      width: 10,
      height: 10,
      posX: 30,
      posY: 14,
      shadow: true,
      color: "#447766",
    },
    {
      id: 5,
      width: 10,
      height: 10,
      posX: 40,
      posY: 34,
      shadow: true,
      color: "#173f32",
    },
    {
      id: 6,
      width: 5,
      height: 5,
      posX: 5,
      posY: 5,
      shadow: true,
      color: "#397e67",
    },
    {
      id: 7,
      width: 10,
      height: 10,
      posX: 20,
      posY: 24,
      shadow: true,
      color: "#225e4a",
    },
    {
      id: 8,
      width: 5,
      height: 5,
      posX: 15,
      posY: 10,
      shadow: true,
      color: "#1c6d52",
    },
    {
      id: 9,
      width: 10,
      height: 10,
      posX: 50,
      posY: 4,
      shadow: true,
      color: "#447766",
    },
    {
      id: 10,
      width: 10,
      height: 10,
      posX: 40,
      posY: 14,
      shadow: true,
      color: "#173f32",
    },
    {
      id: 11,
      width: 10,
      height: 10,
      posX: 10,
      posY: 54,
      shadow: true,
      color: "#397e67",
    },
    {
      id: 12,
      width: 10,
      height: 10,
      posX: 30,
      posY: 24,
      shadow: true,
      color: "#225e4a",
    },
    {
      id: 13,
      width: 10,
      height: 10,
      posX: 20,
      posY: 44,
      shadow: true,
      color: "#1c6d52",
    },
    {
      id: 14,
      width: 10,
      height: 10,
      posX: 10,
      posY: 44,
      shadow: true,
      color: "#447766",
    },
    {
      id: 15,
      width: 10,
      height: 10,
      posX: 40,
      posY: 4,
      shadow: true,
      color: "#397e67",
    },
    {
      id: 16,
      width: 5,
      height: 5,
      posX: 110,
      posY: 40,
      shadow: true,
      color: "#173f32",
    },
    {
      id: 17,
      width: 5,
      height: 5,
      posX: 100,
      posY: 45,
      shadow: true,
      color: "#1c6d52",
    },
    {
      id: 18,
      width: 5,
      height: 5,
      posX: 105,
      posY: 45,
      shadow: true,
      color: "#447766",
    },
    {
      id: 20,
      width: 5,
      height: 5,
      posX: 100,
      posY: 50,
      shadow: true,
      color: "#173f32",
    },
    {
      id: 21,
      width: 5,
      height: 5,
      posX: 95,
      posY: 50,
      shadow: true,
      color: "#397e67",
    },
    {
      id: 22,
      width: 5,
      height: 5,
      posX: 90,
      posY: 45,
      shadow: true,
      color: "#225e4a",
    },
    {
      id: 23,
      width: 5,
      height: 5,
      posX: 90,
      posY: 55,
      shadow: true,
      color: "#1c6d52",
    },
    {
      id: 24,
      width: 5,
      height: 5,
      posX: 110,
      posY: 50,
      shadow: true,
      color: "#447766",
    },
    {
      id: 25,
      width: 4,
      height: 4,
      posX: 2,
      posY: 40,
      shadow: true,
      color: "#173f32",
    },
    {
      id: 26,
      width: 4,
      height: 4,
      posX: 110,
      posY: 15,
      shadow: true,
      color: "#397e67",
    },
    {
      id: 27,
      width: 4,
      height: 4,
      posX: 0,
      posY: 54,
      shadow: true,
      color: "#225e4a",
    },
    {
      id: 28,
      width: 4,
      height: 4,
      posX: 90,
      posY: 5,
      shadow: true,
      color: "#1c6d52",
    },
  ],
};

export const ctaOptions = {
  light: {
    id: 30,
    width: 30,
    height: 7,
    posX: 50,
    posY: 50,
    shadow: true,
    color: "#002a38",
  },
  dark: {
    id: 30,
    width: 30,
    height: 7,
    posX: 50,
    posY: 50,
    shadow: true,
    color: "#DBDBDB",
  },
};
