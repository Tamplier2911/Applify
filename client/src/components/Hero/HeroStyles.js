import styled, { css } from "styled-components";
import { Link } from "react-router-dom";
import bg from "../../assets/jpeg/tabletbg-min.jpg";

import ScrollAnimation from "react-animate-on-scroll";

const fadeInHeroComponent = css`
  @keyframes heroFadeIn {
    0% {
      opacity: 0;
    }
    100% {
      opacity: 1;
    }
  }
`;

export const HeroContainer = styled.div`
  ${fadeInHeroComponent}
  animation: 1s heroFadeIn;

  display: grid;
  position: relative;
  overflow: hidden;
  grid-template-columns: repeat(2, 1fr);

  @media only screen and (max-width: 720px) {
    grid-template-columns: min-content;
    background-image: linear-gradient(to bottom right, #fffffffb, #ffffff2c),
      url(${bg});
    background-position: center;
    background-size: cover;
    background-repeat: no-repeat;
  }

  @media only screen and (max-width: 720px) {
    @supports (-ms-ime-align: auto) {
      background-image: linear-gradient(
          to bottom right,
          rgba(255, 255, 255, 0.9),
          rgba(255, 255, 255, 0.2)
        ),
        url(${bg});
    }
  }
`;

export const HeroActiveCTA = styled.div`
  @media only screen and (max-width: 720px) {
    display: none;
  }
`;

export const HeroActiveCTALink = styled(Link)``;

export const HeroContent = styled.div`
  grid-column: 2 / 3;
  font-family: var(--font-title);

  display: grid;
  grid-template-rows: repeat(2, min-content);
  align-content: center;
  justify-items: center;
  margin-bottom: 10rem;

  @media only screen and (max-width: 980px) {
    margin-bottom: 25rem;
    justify-items: start;
  }

  @media only screen and (max-width: 720px) {
    margin: 0;
    grid-row-gap: 4rem;
    justify-items: center;
  }

  @media only screen and (max-width: 540px) {
    grid-row-gap: 5rem;
  }

  @media only screen and (max-width: 360px) {
    grid-row-gap: 8rem;
  }
`;

export const HeroTextbox = styled(ScrollAnimation)`
  display: grid;
  justify-items: center;
`;

export const HeroHeader = styled.h1`
  font-size: 5rem;
  font-weight: 400;

  @media only screen and (max-width: 980px) {
    font-size: 4rem;
  }

  @media only screen and (max-width: 720px) {
    font-size: 4.2rem;
  }

  @media only screen and (max-width: 300px) {
    font-size: 3.5rem;
    text-align: center;
  }
`;

export const HeroText = styled.h3`
  font-weight: 400;

  @media only screen and (max-width: 980px) {
    font-size: 1.6rem;
  }

  @media only screen and (max-width: 720px) {
    font-size: 1.8rem;
  }

  @media only screen and (max-width: 300px) {
    font-size: 1.7rem;
    text-align: center;
  }
`;

export const HeroPassiveCTA = styled(Link)`
  @media only screen and (min-width: 721px) {
    display: none;
  }

  &:link,
  &:visited {
    text-decoration: none;
    background-color: var(--cl-tiles-2);
    padding: 1rem 5rem;
    color: var(--cl-white);
    font-size: 1.5rem;
    border: 0.1rem solid var(--cl-tiles-2);
    box-shadow: 0 0.2rem 0.4rem var(--cl-shadow);

    transition: color 0.3s, background-color 0.3s;

    @media only screen and (max-width: 540px) {
      padding: 1rem 8rem;
    }

    @media only screen and (max-width: 300px) {
      padding: 1rem 4rem;
    }
  }

  &:hover,
  &:active {
    color: var(--cl-tiles-2);
    background-color: var(--cl-white);
  }
`;
