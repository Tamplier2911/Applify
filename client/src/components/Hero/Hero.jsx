// import "./Hero.scss";
import React from "react";

// redux
import { connect } from "react-redux";
import { createStructuredSelector } from "reselect";
import { selectCurrentLanguage } from "../../redux/lang/lang.selectors";

// components
import Square from "../Square/Square";

// JS Rendering CSS
import {
  HeroContainer,
  HeroActiveCTA,
  HeroActiveCTALink,
  HeroContent,
  HeroTextbox,
  HeroHeader,
  HeroText,
  HeroPassiveCTA,
} from "./HeroStyles";

// component constants
import { squares, ctaOptions } from "./HeroSquaresConstants";
import heroData from "./HeroConstants";

const Hero = ({ lang }) => {
  const currentData = heroData[lang];
  const { companyName, companySlogan, companyCTA } = currentData;
  return (
    <HeroContainer>
      {squares.map((obj) => {
        return <Square key={obj.id} options={obj} />;
      })}
      <HeroActiveCTA>
        <HeroActiveCTALink to="/contacts">
          <Square options={ctaOptions} text={companyCTA} />
        </HeroActiveCTALink>
      </HeroActiveCTA>
      <HeroContent>
        <HeroTextbox animateIn={"bounceInRight"} animateOnce={true}>
          <HeroHeader>{companyName}</HeroHeader>
          <HeroText>{companySlogan}</HeroText>
        </HeroTextbox>
        <HeroPassiveCTA to="/contacts">{companyCTA}</HeroPassiveCTA>
      </HeroContent>
    </HeroContainer>
  );
};

const mapStateToProps = createStructuredSelector({
  lang: selectCurrentLanguage,
});

export default connect(mapStateToProps)(Hero);
