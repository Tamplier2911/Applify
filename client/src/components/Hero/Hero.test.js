import React from "react";

// component
import Hero from "./Hero";

// utils
import {
  setShallowC,
  findByTestAttr,
  storeFactory,
} from "../../tests/testUtils";

describe("<Hero />", () => {
  const initialStore = {
    theme: {
      currentTheme: "light",
    },
    lang: {
      languagePanelHidden: true,
      currentLanguage: "eng",
    },
  };
  const store = storeFactory(initialStore);

  let wrapper;
  beforeEach(() => {
    wrapper = setShallowC(<Hero store={store} />);
  });

  it("renders without an error", () => {
    const hero = findByTestAttr(wrapper, "hero");
    expect(hero.length).toBe(1);
  });

  it("renders children - hero CTA", () => {
    const heroCTA = findByTestAttr(wrapper, "hero-cta");
    expect(heroCTA.length).not.toBe(0);
  });

  it("renders children - hero content", () => {
    const heroContent = findByTestAttr(wrapper, "hero-content");
    expect(heroContent.length).toBe(1);
  });

  it("matches snapshot", () => {
    const hero = findByTestAttr(wrapper, "hero");
    expect(hero).toMatchSnapshot();
  });
});
