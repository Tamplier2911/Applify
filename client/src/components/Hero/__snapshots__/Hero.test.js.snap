// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hero /> matches snapshot 1`] = `
<styled.div
  data-test="hero"
  theme="light"
>
  <Square
    key="1"
    options={
      Object {
        "color": "#fe3f2d",
        "height": 10,
        "id": 1,
        "posX": 10,
        "posY": 24,
        "shadow": true,
        "width": 10,
      }
    }
  />
  <Square
    key="2"
    options={
      Object {
        "color": "#ffaa2d",
        "height": 10,
        "id": 2,
        "posX": 30,
        "posY": 34,
        "shadow": true,
        "width": 10,
      }
    }
  />
  <Square
    key="3"
    options={
      Object {
        "color": "#00a8a9",
        "height": 10,
        "id": 3,
        "posX": 20,
        "posY": 34,
        "shadow": true,
        "width": 10,
      }
    }
  />
  <Square
    key="4"
    options={
      Object {
        "color": "#fe3f2d",
        "height": 10,
        "id": 4,
        "posX": 30,
        "posY": 14,
        "shadow": true,
        "width": 10,
      }
    }
  />
  <Square
    key="5"
    options={
      Object {
        "color": "#fed595",
        "height": 10,
        "id": 5,
        "posX": 40,
        "posY": 34,
        "shadow": true,
        "width": 10,
      }
    }
  />
  <Square
    key="6"
    options={
      Object {
        "color": "#fe3f2d",
        "height": 5,
        "id": 6,
        "posX": 5,
        "posY": 5,
        "shadow": true,
        "width": 5,
      }
    }
  />
  <Square
    key="7"
    options={
      Object {
        "color": "#ffaa2d",
        "height": 10,
        "id": 7,
        "posX": 20,
        "posY": 24,
        "shadow": true,
        "width": 10,
      }
    }
  />
  <Square
    key="8"
    options={
      Object {
        "color": "#00a8a9",
        "height": 5,
        "id": 8,
        "posX": 15,
        "posY": 10,
        "shadow": true,
        "width": 5,
      }
    }
  />
  <Square
    key="9"
    options={
      Object {
        "color": "#002a38",
        "height": 10,
        "id": 9,
        "posX": 50,
        "posY": 4,
        "shadow": true,
        "width": 10,
      }
    }
  />
  <Square
    key="10"
    options={
      Object {
        "color": "#00a8a9",
        "height": 10,
        "id": 10,
        "posX": 40,
        "posY": 14,
        "shadow": true,
        "width": 10,
      }
    }
  />
  <Square
    key="11"
    options={
      Object {
        "color": "#fed595",
        "height": 10,
        "id": 11,
        "posX": 10,
        "posY": 54,
        "shadow": true,
        "width": 10,
      }
    }
  />
  <Square
    key="12"
    options={
      Object {
        "color": "#002a38",
        "height": 10,
        "id": 12,
        "posX": 30,
        "posY": 24,
        "shadow": true,
        "width": 10,
      }
    }
  />
  <Square
    key="13"
    options={
      Object {
        "color": "#fe3f2d",
        "height": 10,
        "id": 13,
        "posX": 20,
        "posY": 44,
        "shadow": true,
        "width": 10,
      }
    }
  />
  <Square
    key="14"
    options={
      Object {
        "color": "#ffaa2d",
        "height": 10,
        "id": 14,
        "posX": 10,
        "posY": 44,
        "shadow": true,
        "width": 10,
      }
    }
  />
  <Square
    key="15"
    options={
      Object {
        "color": "#fed595",
        "height": 10,
        "id": 15,
        "posX": 40,
        "posY": 4,
        "shadow": true,
        "width": 10,
      }
    }
  />
  <Square
    key="16"
    options={
      Object {
        "color": "#d6d7d9",
        "height": 5,
        "id": 16,
        "posX": 110,
        "posY": 40,
        "shadow": true,
        "width": 5,
      }
    }
  />
  <Square
    key="17"
    options={
      Object {
        "color": "#c0c1c3",
        "height": 5,
        "id": 17,
        "posX": 100,
        "posY": 45,
        "shadow": true,
        "width": 5,
      }
    }
  />
  <Square
    key="18"
    options={
      Object {
        "color": "#d6d7d9",
        "height": 5,
        "id": 18,
        "posX": 105,
        "posY": 45,
        "shadow": true,
        "width": 5,
      }
    }
  />
  <Square
    key="20"
    options={
      Object {
        "color": "#bfc0c2",
        "height": 5,
        "id": 20,
        "posX": 100,
        "posY": 50,
        "shadow": true,
        "width": 5,
      }
    }
  />
  <Square
    key="21"
    options={
      Object {
        "color": "#bfc0c2",
        "height": 5,
        "id": 21,
        "posX": 95,
        "posY": 50,
        "shadow": true,
        "width": 5,
      }
    }
  />
  <Square
    key="22"
    options={
      Object {
        "color": "#d6d7d9",
        "height": 5,
        "id": 22,
        "posX": 90,
        "posY": 45,
        "shadow": true,
        "width": 5,
      }
    }
  />
  <Square
    key="23"
    options={
      Object {
        "color": "#bfc0c2",
        "height": 5,
        "id": 23,
        "posX": 90,
        "posY": 55,
        "shadow": true,
        "width": 5,
      }
    }
  />
  <Square
    key="24"
    options={
      Object {
        "color": "#d6d7d9",
        "height": 5,
        "id": 24,
        "posX": 110,
        "posY": 50,
        "shadow": true,
        "width": 5,
      }
    }
  />
  <Square
    key="25"
    options={
      Object {
        "color": "#ffaa2d",
        "height": 4,
        "id": 25,
        "posX": 2,
        "posY": 40,
        "shadow": true,
        "width": 4,
      }
    }
  />
  <Square
    key="26"
    options={
      Object {
        "color": "#fed595",
        "height": 4,
        "id": 26,
        "posX": 110,
        "posY": 15,
        "shadow": true,
        "width": 4,
      }
    }
  />
  <Square
    key="27"
    options={
      Object {
        "color": "#fe3f2d",
        "height": 4,
        "id": 27,
        "posX": 0,
        "posY": 54,
        "shadow": true,
        "width": 4,
      }
    }
  />
  <Square
    key="28"
    options={
      Object {
        "color": "#00a8a9",
        "height": 4,
        "id": 28,
        "posX": 90,
        "posY": 5,
        "shadow": true,
        "width": 4,
      }
    }
  />
  <styled.div
    data-test="hero-cta"
  >
    <Styled(Link)
      to="/contacts"
    >
      <Square
        options={
          Object {
            "color": "#002a38",
            "height": 7,
            "id": 30,
            "posX": 50,
            "posY": 50,
            "shadow": true,
            "width": 30,
          }
        }
        text="Get Started"
      />
    </Styled(Link)>
  </styled.div>
  <styled.div
    data-test="hero-content"
  >
    <Styled(ScrollAnimation)
      animateIn="bounceInRight"
      animateOnce={true}
    >
      <styled.h1>
        Applify
      </styled.h1>
      <styled.h3>
        Creative web solutions for your business...
      </styled.h3>
    </Styled(ScrollAnimation)>
    <Styled(Link)
      data-test="hero-cta"
      theme="light"
      to="/contacts"
    >
      Get Started
    </Styled(Link)>
  </styled.div>
</styled.div>
`;
